cmd/cni-metrics-helper/metrics/cni_metrics_test.go

```diff
@@ -18,33 +22,37 @@ var logConfig = logger.Configuration{
 	LogLocation: "stdout",
 }
 
-var log = logger.New(&logConfig)
+var testLog = logger.New(&logConfig)
 
 type testMocks struct {
-	ctrl               *gomock.Controller
 	clientset          *k8sfake.Clientset
-	discoverController *k8sapi.Controller
+    podWatcher         *defaultPodWatcher
```

Comment:
Nit: Bad indent - Does this mean our gofmt checks aren't working? :confused:

Reply:
Indent is alright in the IDE but off in the diff..will rerun gofmt

Reply:
> Indent is alright in the IDE but off in the diff

Ah, your editor is probably using spaces rather than tabs.  golang likes actual tab chars - the opposite of (eg) python style.